src/Compilers/CSharp/Test/Symbol/Symbols/AnonymousTypesSemanticsTests.cs

```diff
@@ -419,6 +723,33 @@ void m()
         }
 
         [Fact()]
+        public void AnonymousTypeFieldCreatedInLambda_OperationTree()
+        {
+            string source = @"
+using System;
+class ClassA
+{
+    void m()
+    {
+        var o = (Action)(() => (/*<bind>*/new { x = 1, y = new { } }/*</bind>*/).ToString()); ;
+    }
+}
+";
```

Comment:
What is different about anonymous types created in a lambda?

That said, perhaps reference locals or parameters from the containing method, if there are not already tests that cover captured variables, although captured variables can be tested separately from anonymous types.